bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/rating/Rating.java

```diff
@@ -0,0 +1,116 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.form.rating;
+
+import static de.agilecoders.wicket.jquery.JQuery.$;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.wicket.Application;
+import org.apache.wicket.ajax.AjaxEventBehavior;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.markup.head.CssHeaderItem;
+import org.apache.wicket.markup.head.HeaderItem;
+import org.apache.wicket.markup.head.IHeaderResponse;
+import org.apache.wicket.markup.head.JavaScriptHeaderItem;
+import org.apache.wicket.markup.head.OnDomReadyHeaderItem;
+import org.apache.wicket.markup.html.form.HiddenField;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.request.resource.CssResourceReference;
+import org.apache.wicket.request.resource.JavaScriptResourceReference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsCssResourceReference;
+
+/**
+ * Bootstrap Rating is a jQuery plugin that creates a rating control that uses Bootstrap glyphicons for rating symbols. 
+ * http://dreyescat.github.io/bootstrap-rating
+ * @author daniel.jipa
+ */
+public class Rating<T> extends HiddenField<T>{
```

Comment:
I'd prefer to rename it to `RatingField`. Just `Rating` sounds like a POJO/entity to me.
WDYT?